### ISSUE TITLE
NVDA becoming sluggish when my laptop is running on battery

### ISSUE THREAD

**Issue by hozosch** [nvaccess/nvda#13839]
### Steps to reproduce:
laptop needs to be running on battery
energy saving plan needs to be set to balanced
### Actual behavior:
NVDA really becomes sluggish, except on websites and menus like the NVDA menu
### Expected behavior:
NVDA should be as responsive as it is when plugged in, or energy saving plan is set to highest performance
#### NVDA installed/portable/running from source:
installed
#### NVDA version:
2022.1
#### Windows version:
windows 11 home 64 bit (10.0.22621)
#### Name and version of other software in use when reproducing the issue:
none
#### Other information about your system:
We can't blame the insider version here either, because I noticed it on the stable Windows version too.
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Don't think it would make a difference. I started out with 2021.3 I believe, definitely earlier than the current version anyway.
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by zstanecic** [comment]
Hi!

Which laptop is in question?

It needs more details for investigation.

 

From: hozosch ***@***.***> 
Sent: Thursday, June 23, 2022 9:56 PM
To: nvaccess/nvda ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [nvaccess/nvda] NVDA becoming sluggish when my laptop is running on battery (Issue #13839)

 


Steps to reproduce:


laptop needs to be running on battery
energy saving plan needs to be set to balanced


Actual behavior:


NVDA really becomes sluggish, except on websites and menus like the NVDA menu


Expected behavior:


NVDA should be as responsive as it is when plugged in, or energy saving plan is set to highest performance


System configuration


NVDA installed/portable/running from source:


installed


NVDA version:


2022.1


Windows version:


windows 11 home 64 bit (10.0.22621)


Name and version of other software in use when reproducing the issue:


none


Other information about your system:


We can't blame the insider version here either, because I noticed it on the stable Windows version too.


Other questions


Does the issue still occur after restarting your computer?


Yes


Have you tried any other versions of NVDA? If so, please report their behaviors.


Don't think it would make a difference. I started out with 2021.3 I believe, definitely earlier than the current version anyway.


If NVDA add-ons are disabled, is your problem still occurring?


Yes


Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?


Yes

**Comment by hozosch** [comment]
Yes, i realized I didn't put enough details in here shortly after I sent this. What I can say now that this machine is by the Medion company. What else do you think is important for you to know? I'm not sure what to pay attention to here.

**Comment by Brian1Gaff** [comment]
Processor, hard drive size, and type, ie ssd or not, graphics hardware, and 
does it nick system memory?
memory fitted.
 Does this happen on off line software or merely on the web? If the latter 
and its quite slow and maybe low in memory, when its throttled back to save 
power, a lot of Javascript rich pages can be completely unusable.
 It obviously has something to do with resources and throttling back, so its 
far more likely to be machine running out of resources than NVDA. Can you at 
least temporarily run it at the full speed on battery? OK it will impact the 
battery life, but seems to me that tweaking the settings is the way to go, 
or get a more capable machine with a bigger battery!

Brian

-- 
***@***.***
Sent via blueyonder.(Virgin media)
Please address personal E-mail to:-
***@***.***, putting 'Brian Gaff'
in the display name field.
----- Original Message -----

**Comment by hozosch** [comment]
Ok, maybe you are right, you can't do anything in NVDA about it. But I'm not sure, because when I press a key which I'm not meant to press, which causes the Windows Ding sound to play, it responds immediately, but NVDA is very unresponsive in the balanced energy saving plan, which I already notice when just tabbing through the task bar.
Anyway, here are some of the details:
Processor	Intel(R) Core(TM) i3-10110U CPU @ 2.10GHz, 2501 MHz, 2 core(s), 4 logical Processor(s)
It's using Intel UHD graphics, and a 250 GB SSd, 8 GB ram.
I'm not sure about the other things you asked. If you mean how much ram it uses, currently about 3 Gb are still free.

**Comment by seanbudd** [comment]
This is not a problem we can fix.
NVDA can make general performance improvements over time, but this issue is to broad. You could improve the performance while on battery, however of course your battery will run out sooner: https://docs.microsoft.com/en-us/windows-hardware/customize/desktop/customize-power-slider
